contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -0,0 +1,166 @@
+end
+
+#
+# Getters
+#
+
+@view
+func name{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (name : felt):
+    let (name) = ERC20_name()
+    return (name)
+end
+
+@view
+func symbol{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (symbol : felt):
+    let (symbol) = ERC20_symbol()
+    return (symbol)
+end
+
+@view
+func totalSupply{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+        totalSupply : Uint256):
+    let (totalSupply : Uint256) = ERC20_totalSupply()
+    return (totalSupply)
+end
+
+@view
+func decimals{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+        decimals : felt):
+    let (decimals) = ERC20_decimals()
+    return (decimals)
+end
+
+@view
+func balanceOf{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        account : felt) -> (balance : Uint256):
+    let (balance : Uint256) = ERC20_balanceOf(account)
+    return (balance)
+end
+
+@view
+func allowance{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        owner : felt, spender : felt) -> (remaining : Uint256):
+    let (remaining : Uint256) = ERC20_allowance(owner, spender)
+    return (remaining)
+end
+
+#
+# Externals
+#
+
+@external
+func transfer{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        recipient : felt, amount : Uint256) -> (success : felt):
+    let (from_) = get_caller_address()
+    claimable_before_token_transfer(from_, recipient)
+    ERC20_transfer(recipient, amount)
+    return (TRUE)
+end
+
+@external
+func transferFrom{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        sender : felt, recipient : felt, amount : Uint256) -> (success : felt):
+    claimable_before_token_transfer(sender, recipient)
+    ERC20_transferFrom(sender, recipient, amount)
+    return (TRUE)
+end
+
+@external
+func approve{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        spender : felt, amount : Uint256) -> (success : felt):
+    ERC20_approve(spender, amount)
+    return (TRUE)
+end
+
+@external
+func increaseAllowance{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        spender : felt, added_value : Uint256) -> (success : felt):
+    ERC20_increaseAllowance(spender, added_value)
+    return (TRUE)
+end
+
+@external
+func decreaseAllowance{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        spender : felt, subtracted_value : Uint256) -> (success : felt):
+    ERC20_decreaseAllowance(spender, subtracted_value)
+    return (TRUE)
+end
+
+@external
+func mint{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+        recipient : felt, amount : Uint256):
+    Ownable_only_owner()
+    claimable_before_token_transfer(0, recipient)
+    ERC20_mint(recipient, amount)
+    return ()
+end
+
+@external
+func claimRewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
```

Comment:
general question: shouldn't we use snake case for cairo?

Reply:
Yes, will fix this everywhere